### DISCUSSION TITLE
Type error only when building

### DISCUSSION THREAD

**Discussion by laurent512** [prisma/prisma#17078] [Q&A]
Hello, 

To get access to the return type of a prisma function, I used the solution described at the very bottom in the documentation here : https://www.prisma.io/docs/concepts/components/prisma-client/advanced-type-safety/operating-against-partial-structures-of-model-types  : 
```
import { Prisma } from '@prisma/client'
type UsersWithPosts = Prisma.PromiseReturnType<typeof getUsersWithPosts>
```
After fetching the data I wanted ( with my own schema here)  : 
 const reccurentExpenses: RecurrentExpenseWithRecurrency = await getData()

After that, I would like to itterate through the objects array fetched using {reccurentExpenses.map((expense) => ())}
This works perfectly in development mode (npm run dev) but when I try to deploy using vercel, it does not work anymore. 
It tells me that " Type error: Parameter 'expense' implicitly has an 'any' type." 

Do you have any idea why this could occur ? 

My tsconfig is the following : 
```
{
    "compilerOptions": {
        "target": "es6",
        "lib": ["dom", "dom.iterable", "esnext"],
        "allowJs": true,
        "skipLibCheck": true,
        "strict": true,
        "forceConsistentCasingInFileNames": true,
        "noEmit": true,
        "esModuleInterop": true,
        "module": "esnext",
        "moduleResolution": "node",
        "resolveJsonModule": true,
        "isolatedModules": true,
        "jsx": "preserve",
        "incremental": true,
        "plugins": [
            {
                "name": "next"
            }
        ],
        "baseUrl": "./",
        "paths": {
            "@components/*": ["components/*"],
            "@contentful/*": ["contentful/*"],
            "@styles/*": ["styles/*"],
            "@utils/*": ["util/*"],
            "@images/*": ["/public/images/*"],
            "@shared/*": ["shared/*"],
            "@types/*": ["types/*"]
        }
    },
    "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
    "exclude": ["node_modules"]
}

```

**Comment by nurul3101** [comment]
Hello @laurent512 👋 

Are you getting valid data in `reccurentExpenses` in your production database?
Can you try logging it and check if it is of same format as your development database?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
